README.md

```diff
@@ -163,11 +184,66 @@ docker exec -i maraudersmap-pg /bin/bash -c "PGPASSWORD=password psql --username
 > assumes names and passwords as given in `docker run` above.
 
 #### Update `seed.sql`
-If you want to add more data to the sample, simply dump your current database. From project root, run
+If you want to update the sample data, simply replace the file with a dump of your current database
 ```
 make dump
 ```
 or
 ```
 docker exec -i maraudersmap-pg /bin/bash -c "PGPASSWORD=password pg_dump --username postgres postgres" > maraudersmap/database/seed.sql
 ```
+
+### 💣️ Delete database
+The database can be deleted with
+```
+make nukedb
+```
+or
+```
+docker stop maraudersmap-pg
+docker rm maraudersmap-pg
+docker volume rm maraudersmap-data
+```
+
+## 🌐 Deployment (with Fly.io)
+The tile server can be deployed in a lot of places. We currently use [Fly.io](https://fly.io/) for quick prototyping.
```

Comment:
Should we briefly explain what a tile server is or is it obvious?

Reply:
Yes, that's not obvious, will add :+1:

Reply:
Wait, this is the wrong repo... This is a copy-paste error 🙃

Added an explanation in the actual tile server repo: https://github.com/heyloft/maraudersmap-tiles/pull/1#discussion_r1018832645